tests/data/suite.json

```diff
@@ -22,7 +22,7 @@
     "skipped_tests": [
       "LibTiffInfoTest"
     ],
-    "status": "passed"
+    "status": "GREEN"
```

Comment:
Did you update this manually or run the `tests/data/generate.py` script? If the former, I suggest running the script to ensure that the files are generated consistently. It also acts as a bit of a test. 

Actually, it might make sense to convert the code in `generate.py` to Pytest fixtures, which generate the data files dynamically during testing. This way, it's automatic. This would be a good future refactoring task.

Reply:
I did this manually originally. The latest commit adds minor changes from running `generate.py` and adds a TODO to move that logic to pytest fictures.